File: Assets/Plugins/LookAt.js
#pragma strict
var target: GameObject;
var up: Vector3 = new Vector3(0, 1, 0);
function Awake() { 
    Debug.Log("Look at Awake() called");
    target = GameObject.FindWithTag('Enemy'); }
function Update () {
	if(null != target) { transform.LookAt(target.transform, up); }
}

File: Assets/Script/LookAt.cs
using UnityEngine;
public class LookAt : MonoBehaviour {
	public GameObject target;
	private GameObject me;
	private Vector3 up = new Vector3(0, 1, 0);
	public void Awake() { 
		target = GameObject.FindWithTag("Enemy");
	}
	public void Update () {
		if(null != target) { transform.LookAt(target.transform, up); }
	}
}
